README.md

```diff
@@ -1,13 +1,18 @@
 # Rails Engine Documentation
 
-This application creates an API for a series of merchants and their transactions, invoices, items, and customers.
+Rails Engine is a public-facing API which serves business intelligence via JSON. It's recommended that you use a browser extension such as [JSONView](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc) to prettify all that sweet, sweet JSON. 
 
 This application uses:
   * Ruby version 2.3.3
   * Rails version 5.1.2
   * PostgreSQL version 9.6.1
 
-To request an API endpoint, prepend 'https://cody-nick-rails-engine.herokuapp.com' before the endpoint path in your browser's URL bar and send the request. Your browser will then receive data as a JSON object. The following endpoints are available:
+Once you've cloned this repository, you'll need to run the following commands in your terminal:
+
+* rails db:setup - to create the database and prepare the tables & columns according to the schema
+* rake csv:import - to import the sales data via rake task
+
+To request an API endpoint, prepend http://198.199.107.127/ before the endpoint path in your browser's URL bar and send the request. Your browser will then receive data as a JSON object. The following endpoints are available:
```

Comment:
Nice, @codyborders! Your DigitalOcean server is way way faster than the previous Heroku server. This is great.